File: Codewars_C/ex03/t_a_p.c
int points(const char const games[10])
{
    int points = 0; 

    for(int i = 0; i < 10; i++)
    {
        int x,y; 
        sscanf(games[i],"%d:%d",&x,&y);
        if (x > y)
        {
            points += 3;
        }else if (x == y){
            points += 1;
        }else{
            points += 0;
        }
    }
    return points;
}

File: Codewars_Python/8kyu/ex03/t_a_p.py
def points(games):
    points = 0

    for game in games:
        x, y = map(int, game.split(':'))
        if x > y:
            points += 3
        elif x == y:
            points += 1
        # Si x < y, no se suma ningún punto (points += 0)

    return points